src/library-authoring/library-team/LibraryTeam.tsx

```diff
@@ -65,8 +65,13 @@ const LibraryTeam: React.FC<Record<never, never>> = () => {
         accessLevel: LibraryRole.Reader.toString() as LibraryAccessLevel,
       }).then(() => {
         showToast(intl.formatMessage(messages.addMemberSuccess));
-      }).catch(() => {
-        showToast(intl.formatMessage(messages.addMemberError));
+      }).catch((addMemberError) => {
+        const errorData = addMemberError.response.data;
+        if ('email' in errorData) {
```

Comment:
This code is fragile, if a non-Axios error is raised:
![Screenshot 2024-12-04 at 10 12 56 AM](https://github.com/user-attachments/assets/b68f2d96-057b-4d69-a309-af624b1543ad)

Try something like:

```suggestion
        const errorData = typeof addMemberError === 'object' ? addMemberError.response?.data : undefined;
        if (errorData && 'email' in errorData) {
```

Reply:
Updated https://github.com/openedx/frontend-app-authoring/pull/1543/commits/bb90f0bb4f22b16135fb36dbcd1184565e01517d